tests/test_wakatime_formatter.py

```
@@ -0,0 +1,178 @@
+        if method.startswith("test_")
+    ]
+
+    passed = 0
+    failed = 0
+
+    for test_name in test_methods:
+        try:
+            getattr(test_suite, test_name)()
+            print(f"✓ {test_name}")
+            passed += 1
+        except AssertionError as e:
```

Ruff (0.14.10)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)